src/Raven.Server/Documents/Handlers/Admin/Processors/Revisions/AbstractAdminRevisionsHandlerProcessorForEnforceRevisionsConfiguration.cs

```diff
@@ -2,34 +2,42 @@
 using JetBrains.Annotations;
 using Raven.Client.Documents.Operations.Revisions;
 using Raven.Server.Documents.Handlers.Processors;
+using Raven.Server.Documents.Operations;
 using Raven.Server.Json;
 using Raven.Server.ServerWide;
 using Sparrow.Json;
 
 namespace Raven.Server.Documents.Handlers.Admin.Processors.Revisions
 {
-    internal abstract class AbstractAdminRevisionsHandlerProcessorForEnforceRevisionsConfiguration<TRequestHandler, TOperationContext> : AbstractDatabaseHandlerProcessor<TRequestHandler, TOperationContext>
+    internal abstract class AbstractAdminRevisionsHandlerProcessorForRevisionsOperation<TRequestHandler, TOperationContext, TOperationParameters> : AbstractDatabaseHandlerProcessor<TRequestHandler, TOperationContext>
         where TOperationContext : JsonOperationContext
         where TRequestHandler : AbstractDatabaseRequestHandler<TOperationContext>
+        where TOperationParameters : ReveisionsOperationParameters
     {
-        protected AbstractAdminRevisionsHandlerProcessorForEnforceRevisionsConfiguration([NotNull] TRequestHandler requestHandler) : base(requestHandler)
+        protected OperationType _operationType;
+        public string Description { get; set; } = string.Empty;
```

Comment:
either have a proper text or don't implement this here.

Reply:
Done